### PR TITLE
fix(SafeArea): Ensure the root of the XAML tree is used to compute relative bounds of SafeArea control/owner

### PR DESCRIPTION
**Pull request by kazo0**
closes https://github.com/unoplatform/nventive-private/issues/412

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

SafeArea calculates improper offsets when the control is within a native iOS Modal window. We were using `TransformToVisual` relative to `Windows.Current.Content` which results in an offset of X=16, Y=47 since the Window.Content (white background) is no longer at position 0,0 while a modal (red background) is being displayed

MicrosoftTeams-image (2)

## What is the new behavior?

Use `TransformToVisual(null)` to ensure that the root of the XAML tree is used for the transformation

### CODE REVIEW

File: src/Uno.Toolkit.UI/Controls/SafeArea/SafeArea.cs [hunk 1]
```diff
@@ -354,7 +354,9 @@ private void UpdateInsets()
 					// If the owner view is scrollable, the visibility of interest is that of the scroll viewport.
 					var fixedControl = scrollAncestor ?? Owner;
 
-					var controlBounds = GetRelativeBounds(fixedControl, XamlWindow.Current.Content);
+					// Using relativeTo: null instead of Window.Current.Content since there are cases when the current UIElement
```
Comment (jeromelaban) [inline src/Uno.Toolkit.UI/Controls/SafeArea/SafeArea.cs:357]: Is there a way to UI Test this ?

Comment (kazo0) [reply src/Uno.Toolkit.UI/Controls/SafeArea/SafeArea.cs:357]: Added!

File: samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Droid/Uno.Toolkit.Samples.Droid.csproj [hunk 2]
```diff
@@ -16,7 +16,6 @@
     <AndroidResgenFile>Resources\Resource.Designer.cs</AndroidResgenFile>
     <AndroidUseAapt2>true</AndroidUseAapt2>
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
-    <AndroidUseLatestPlatformSdk>False</AndroidUseLatestPlatformSdk>
```
Comment (jeromelaban) [inline samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Droid/Uno.Toolkit.Samples.Droid.csproj:19]: Why did you need this ?

**Review by jeromelaban — APPROVED** [review]
